contracts/registries/EthFeeRegistry.sol

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright ©️ 2018-2020 Galt•Project Society Construction and Terraforming Company
+ * (Founded by [Nikolai Popeka](https://github.com/npopeka)
+ *
+ * Copyright ©️ 2018-2020 Galt•Core Blockchain Company
+ * (Founded by [Nikolai Popeka](https://github.com/npopeka) by
+ * [Basic Agreement](ipfs/QmaCiXUmSrP16Gz8Jdzq6AJESY1EAANmmwha15uR3c1bsS)).
+ */
+
+pragma solidity ^0.5.13;
+
+import "@galtproject/libs/contracts/traits/OwnableAndInitializable.sol";
+import "../interfaces/IEthFeeRegistry.sol";
+
+
+/**
+ * @title Eth Fee Registry.
+ * @notice Tracks all fees.
+ */
+contract EthFeeRegistry is IEthFeeRegistry, OwnableAndInitializable {
+
+  uint256 public constant VERSION = 1;
+
+  mapping(bytes32 => uint256) public ethFeeByKey;
+
+  address public feeManager;
```

Comment:
Why don't use ACL roles? Using explicitly set roles along with ACL is confusing.

Reply:
I don't think that we need many fee managers or many fee collectors, so don't need to use ACL